.env.example

```diff
@@ -0,0 +1,24 @@
+# Jackson settings
+SAML_AUDIENCE=https://saml.boxyhq.com
+JACKSON_API_KEYS="secret"
+IDP_ENABLED=
```

Comment:
Is this treated as an empty string or as though IDP_ENABLED is not defined?

Reply:
It is treated as an empty string `""`